org.lflang/src/org/lflang/generator/c/CCompiler.java

```diff
@@ -210,7 +345,7 @@ public LFCommand compileCCommand(
 
         // If there is no main reactor, then use the -c flag to prevent linking from occurring.
         // FIXME: we could add a `-c` flag to `lfc` to make this explicit in stand-alone mode.
-        // Then again, I think this only makes sense when we can do linking.
+        //  Then again, I think this only makes sense when we can do linking.
```

Comment:
Does this need a second look as suggested by the old FIXME? Or else, can it be removed?

Reply:
As far as I know, the "produce an object file instead of an executable" option does not even work -- I have not seen tests that use it. Therefore, my guess is that this FIXME is still unresolved. I personally do not see why we would want to examine questions related to that compilation option in the near future; given the way connections are resolved at compile time, it is unclear to me whether it ever will be feasible to make it work.